ansible/roles/images/tasks/load.yaml

```diff
@@ -9,12 +9,28 @@
   register: images_cache_find_result
   changed_when: false
 
-- name: import images from files in the local images cache  # noqa no-changed-when
-  shell:
-    executable: /bin/bash
-    cmd: ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }})
+- block:
+    - name: import images from files in the local images cache # noqa no-changed-when
+      shell:
+        executable: /bin/bash
+        cmd: ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }})
+      when:
+        - item.split('/')[-1] | regex_search('^1\.(\d)+\.(\d)+_images\.tar\.gz$')
```

Comment:
There are multiple ways to support backward compatibility.  below are the alternatives i am thinking.
wdyt ?
```yaml
when: "{{ kubernetes_version is version('1.24.4, '>=', version_type='semver') }}"
```
https://github.com/mesosphere/konvoy-image-builder/blob/main/images/common.yaml#L2
https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#comparing-versions

OR 
- try both ways to upload images
- ignore errors
- add new task to fail if `ctr_images_import.rc !=0 && mindthegap.rc !=0`
I am trying above to see if its possible.